tests/resources/Vsphere-Util.robot

```diff
@@ -203,6 +203,7 @@ Add Host To VCenter
     \   ${out}=  Run  govc cluster.add -hostname=${host} -username=${user} -dc=${dc} -password=${pw} -noverify=true
     \   ${status}=  Run Keyword And Return Status  Should Contain  ${out}  OK
     \   Return From Keyword If  ${status}
+    \   Sleep  3 minutes
```

Comment:
I'm curious because we see this particular error only with the `5-5-Heterogenous-ESXi` test. Could this be occurring due to the specific environment in this test? Also pinging @dougm for input.